### PR TITLE
Metrics for measuring verify time for commit votes and decision

### PR DESCRIPTION
**Pull request by vusirikala**
## Description
Metrics for measuring verify time for commit votes and decision

## Type of Change
- [ ] New feature
- [ ] Bug fix
- [ ] Breaking change
- [ ] Performance improvement
- [ ] Refactoring
- [ ] Dependency update
- [ ] Documentation update
- [ ] Tests

## Which Components or Systems Does This Change Impact?
- [ ] Validator Node
- [ ] Full Node (API, Indexer, etc.)
- [ ] Move/Aptos Virtual Machine
- [ ] Aptos Framework
- [ ] Aptos CLI/SDK
- [ ] Developer Infrastructure
- [ ] Other (specify)

## Checklist
- [ ] I have read and followed the [CONTRIBUTING](https://github.com/aptos-labs/aptos-core/blob/main/CONTRIBUTING.md) doc
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I identified and added all stakeholders and component owners affected by this change as reviewers
- [ ] I tested both happy and unhappy path of the functionality
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: consensus/src/pipeline/commit_reliable_broadcast.rs [hunk 1]
```diff
@@ -36,8 +36,22 @@ impl CommitMessage {
     /// Verify the signatures on the message
     pub fn verify(&self, verifier: &ValidatorVerifier) -> anyhow::Result<()> {
         match self {
-            CommitMessage::Vote(vote) => vote.verify(verifier),
-            CommitMessage::Decision(decision) => decision.verify(verifier),
+            CommitMessage::Vote(vote) => {
+                let start_time = Instant::now();
```
Comment (msmouse) [inline consensus/src/pipeline/commit_reliable_broadcast.rs:40]: doesn't look like you need to `observe()` a manually measured duration in this case. just do 
```
{
    let _timer = counters::VERIFY_MSG.timer_with(&[commit_vote]);
    vote.verify(verifier)
}
```

**Review by msmouse — COMMENTED** [review]
would be nice if the summary is cleaned up.

File: consensus/src/pipeline/commit_reliable_broadcast.rs (hunk 1 above)
Comment (vusirikala) [reply consensus/src/pipeline/commit_reliable_broadcast.rs]: Done.

**Review by msmouse — APPROVED** [review]

File: consensus/src/pipeline/commit_reliable_broadcast.rs (hunk 1 above)
Comment (msmouse) [reply consensus/src/pipeline/commit_reliable_broadcast.rs]: saves two lines if you use `.timer_with()`, but your choice 😁

**Review by bchocho — APPROVED** [review]

**Pull request merged**